NineChronicles.Headless/GraphTypes/ActionQuery.cs

```diff
@@ -545,6 +546,8 @@ public ActionQuery(StandaloneContext standaloneContext)
             RegisterRapidCombination();
             RegisterCombinationConsumable();
 
+            base.AddField(new PetEnhancementFieldType(Encode));
```

Comment:
@moreal This line calls the parent method. And this line is in the constructor of `ActionQuery`. So I need to turn off the warning for 'SA1100'.

Reply:
If I change this line like below, another warning was occurred.

```suggestion
            AddField(new PetEnhancementFieldType(Encode));
```
 "Virtual member call in constructor"

Reply:
When I reverted it, it seems worked to me. You can see the changes and the build result at https://github.com/moreal/NineChronicles.Headless/commit/e187a82f8ed54a61adb4e8bf63a0dbe5402a1ae3 commit. Is there something I missed? 🙏🏻

Reply:
@moreal That's little bit weird.🙃 I'll retry now.

Reply:
@moreal The warning message above comes from `ReSharper` linter.. I reverted it. Thanks!